Submodels/Legacy/Bezier_Only_Parameterizations.py

```
@@ -0,0 +1,670 @@
+        u : float
+            Parameter ranging from 0 to 1.
+
+        Returns
+        -------
+        y : float
+            Value of the Bezier curve at parameter u.
+        """
+
+        #Input checking
+        if len(coeff) != 4:
+            raise ValueError(f"Coefficient list must contain exactly 4 elements. Coefficient list contains {len(coeff)} elements")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)